content/source/docs/enterprise/api/modules.html.md

```diff
@@ -311,20 +372,33 @@ After the registry module version is successfully parsed by TFE, its status will
 
 ## Delete a Module
 
-These endpoints can delete a single version for a provider, a single provider (and all its versions) for a module, or a whole module. If the requested deletion would leave a provider with no versions or a module with no providers, the empty items will be automatically deleted as well.
+When removing modules, there are three versions of the endpoint, depending on how many parameters are specified.
+
+* If all parameters (module, provider, and version) are specified, the provided version for the given provider of the module is deleted.
+* If module and provider are specified, the given provider for the module is deleted along with all its versions.
+* If only module is specified, the entire module is deleted.
+
+If a version deletion would leave a provider with no versions, the provider will be deleted. If a provider deletion would leave a module with no providers, the module will be deleted.
 
-| Method | Path |
-| :-- | :-- |
-| `POST` | `/registry-modules/actions/delete/:organization/:module/:provider/:version` |
-| `POST` | `/registry-modules/actions/delete/:organization/:module/:provider` |
-| `POST` | `/registry-modules/actions/delete/:organization/:module` |
+`POST /registry-modules/actions/delete/:organization_name/:name/:provider/:version`
+`POST /registry-modules/actions/delete/:organization_name/:name/:provider`
+`POST /registry-modules/actions/delete/:organization_name/:name`
```

Comment:
UGH, wait, sorry, I forgot: these are going to appear as a single run-together paragraph, because Markdown will see them as a hard wrap. Gotta fix that somehow. The options are:

- Put blank lines between 
- Convert to bullet list
- Add `<br/>` tags at ends of lines
- Convert to code block instead of code spans

I lean toward blank lines because it's the simplest approach, though I admit that `<br>` tags would look slightly nicer.

Reply:
Ah yes sorry, I was intending a list, I think. This is a mistake I make consistently in Markdown.